src/taipy/core/taipy.py

```diff
@@ -74,10 +74,17 @@ def submit(
     Parameters:
         entity (Union[Scenario^, Pipeline^, Task^]): The entity to submit.
         force (bool): If True, the execution is forced even if the data nodes are in cache.
-        wait (bool): Wait for the scheduled jobs created from the submission to be finished in asynchronous mode.
-        timeout (Union[float, int]): The optional maximum number of seconds to wait for the jobs to be finished before
-            returning.
+        wait (bool): Wait for the scheduled jobs created from the submission to be finished
+            in asynchronous mode.
+        timeout (Union[float, int]): The optional maximum number of seconds to wait
+            for the jobs to be finished before returning.
+    Returns:
+        The created `Job^` or a collection of the created `Job^` depends on the submitted entity.
 
+        - If a `Scenario^` is provided, it will return a dictionary with `PipelineId^` as the key and
+            a list of `Job^` corresponding to that pipeline as its value.
+        - If a `Pipeline^` is provided, it will return a list of `Job^`.
```

Comment:
Would it be confusing as it depends on the input entity, we will return different type of objects?

Reply:
Yes. That was under the assumption we return only a list. but you raised a good point about retrieving the pipeline. Let s discuss this in a daily meeting.

Reply:
This has been changed to:
"""
        - If a `Scenario^` or a `Pipeline^` is provided, it will return a list of `Job^`.
        - If a `Task^` is provided, it will return the created `Job^`.
"""